postgresql-plugin/src/test/java/io/cdap/plugin/postgres/PostgresSinkTestRun.java

```diff
@@ -53,6 +59,8 @@
  * Test for ETL using databases.
  */
 public class PostgresSinkTestRun extends PostgresPluginTestBase {
+  private static final String CLASS_NAME = "org.postgresql.Driver";
+  private static final String URL = "jdbc:postgresql://localhost:5432/demodb";
```

Comment:
why this is added, we are suppose to write junits only

Reply:
Working on making the changes accordingly.

Reply:
resolved